packages/ui/tailwind.config.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
-  // purge: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
-  darkMode: 'media', // or 'media' or 'class'
+  purge: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
```

Comment:
Since we are using tailwind v3.0 we should update this line
```suggestion
  content: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
```